pkg/webhooks/github/actions/release-drafter/append_merged_pr.go

```diff
@@ -0,0 +1,111 @@
+package release_drafter
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	"github.com/metal-stack/metal-robot/pkg/clients"
+	"github.com/metal-stack/metal-robot/pkg/markdown"
+	"github.com/metal-stack/metal-robot/pkg/webhooks/github/actions"
+)
+
+type appendMergedPR struct {
+	rd *releaseDrafter
+}
+
+type AppendMergedPrParams struct {
+	Params
+
+	Title  string
+	Number int
+	Author string
+}
+
+func NewAppendMergedPRs(logger *slog.Logger, client *clients.Github, rawConfig map[string]any) (actions.WebhookHandler[*AppendMergedPrParams], error) {
+	rd, err := New(logger, client, rawConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	return &appendMergedPR{
+		rd: rd.(*releaseDrafter),
+	}, nil
+}
+
+func (r *appendMergedPR) Handle(ctx context.Context, p *AppendMergedPrParams) error {
+	_, ok := r.rd.repoMap[p.RepositoryName]
+	if ok {
```

Comment:
We can also consider removing this if-block such that code blocks with special instructions are only considered for repos in the release vector and when the code blocks are in the release notes of the components. This makes it harder for the maintainers because the have to copy the code blocks from their pull requests to their releases though. I am not sure.